internal/testing/runchecks.sh

```diff
@@ -74,9 +73,8 @@ fi
 # Run Go tests for the root. Only do coverage for the Linux build
 # because it is slow, and codecov will only save the last one anyway.
 result=0
-echo
-echo "Running Go tests..."
-if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+if [[ "${TRAVIS_OS_NAME:-}" == "linux" ]]; then
+  echo "Running Go tests (with coverage)..."
```

Comment:
Nit: since we don't have -x now, I'd put a bit more details for identification here: Running Go tests in root directory / Running Go tests in '.' / Running Go tests in the root module, or some such